statsmodels/tsa/vector_ar/irf.py

```diff
@@ -457,8 +457,8 @@ def err_band_sz3(self, orth=False, svar=False, repl=1000, signif=0.05,
         periods = self.periods
         irfs = self._choose_irfs(orth, svar)
         neqs = self.neqs
-        irf_resim = model.irf_resim(orth=orth, repl=repl, T=periods, seed=seed,
-                                   burn=100)
+        irf_resim = model.irf_resim(orth=orth, repl=repl, horizon=periods, seed=seed,
```

Comment:
I think all uses of irf_resim are here.  Should that just be an irf method instead of a VARResults method?

Reply:
Probably, but saving for irf clean